scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/test_hidden_deleted_pins.yaml

```diff
@@ -0,0 +1,150 @@
+# hidden and deleted pins are defined and described by IPC 7351B naming convention
+# http://ohm.bu.edu/~pbohn/__Engineering_Reference/pcb_layout/pcbmatrix/IPC-7x51%20&%20PCBM%20Land%20Pattern%20Naming%20Convention.pdf
+# hidden pins remove a pin location and pin number (for example, going from pad 1 to pad 3 with pin 2 missing from the package)
+# deleted pins remove a pin location but not pin number (for example, pin 2 is missing from the package but the footprint has pad 2)
+# deleted pins do not support custom numbering schemes https://github.com/pointhi/kicad-footprint-generator/pull/371
+
+FileHeader:
+  library_Suffix: 'TO_SOT_SMD'
+  device_type: 'SOT'
+
+# test add_quad_pad_border() in quad_dual_pad_border.py
+LQFP-48_7x7mm_P0.5mm:
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/ltc2358-16.pdf'
+  body_size_x: 7
+  body_size_y: 7
+  overall_size_x: 9
+  overall_size_y: 9
+  lead_width: 0.17 .. 0.27
+  lead_len: 0.45 .. 0.75
+  pitch: 0.5
+  num_pins_x: 12
+  num_pins_y: 12
+
+# test add_quad_pad_border() in quad_dual_pad_border.py with hidden pins
+LQFP-48_7x7mm_P0.5mm_hidden:
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/ltc2358-16.pdf'
+  body_size_x: 7
+  body_size_y: 7
+  overall_size_x: 9
+  overall_size_y: 9
+  lead_width: 0.17 .. 0.27
+  lead_len: 0.45 .. 0.75
+  pitch: 0.5
+  num_pins_x: 12
+  num_pins_y: 12
+  hidden_pins: [1,15,24,30,31,44]
+
+# test add_quad_pad_border() in quad_dual_pad_border.py with deleted pins
+LQFP-48_7x7mm_P0.5mm_deleted:
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/ltc2358-16.pdf'
+  body_size_x: 7
+  body_size_y: 7
+  overall_size_x: 9
+  overall_size_y: 9
+  lead_width: 0.17 .. 0.27
+  lead_len: 0.45 .. 0.75
+  pitch: 0.5
+  num_pins_x: 12
+  num_pins_y: 12
+  deleted_pins: [1,15,24,30,31,44]
+
+# test add_dual_pad_border_y() in quad_dual_pad_border.py
+SOT-23-6:
+  size_source: 'https://www.jedec.org/sites/default/files/docs/Mo-178c.PDF variant AA'
+  custom_name_format: 'SOT-23-6R_Hidden'
```

Comment:
why `_Hidden` if this test-case has no hidden pins.

Reply:
Whoops. Copy and paste. Good catch.